From 0b2572a5b411ebf4295d5a8e17562da086190ad9 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Sun, 19 Aug 2018 01:38:02 +0300
Subject: [PATCH 1/1] Update to 1.2.3. Make some QoL changes to our shipped
 version of coin.

- No, I don't want you going below all my other windows.
- Yes, I do in fact want my theme to apply styling the window.
---
 meson.build         | 8 --------
 src/MainWindow.vala | 4 ----
 2 files changed, 12 deletions(-)

diff --git a/meson.build b/meson.build
index 549d5b2..b40b3a8 100644
--- a/meson.build
+++ b/meson.build
@@ -7,13 +7,6 @@ project('com.github.lainsce.coin', ['vala', 'c'],
 gnome = import('gnome')
 i18n = import('i18n')
 
-# Compile the GResources for this project
-asresources = gnome.compile_resources(
-    'as-resources', 'data/style.gresource.xml',
-    source_dir: 'data',
-    c_name: 'as'
-)
-
 # Set this for convenience
 conf = configuration_data()
 
@@ -45,7 +38,6 @@ executable(
     'src/Application.vala',
     'src/MainWindow.vala',
     'src/Constants/AppSettings.vala',
-    asresources,
     c_args: c_args,
     dependencies: [
         dependency('granite'),
diff --git a/src/MainWindow.vala b/src/MainWindow.vala
index 05dbc5c..cc7cdae 100644
--- a/src/MainWindow.vala
+++ b/src/MainWindow.vala
@@ -46,10 +46,6 @@ namespace Coin {
         }
 
         construct {
-            var provider = new Gtk.CssProvider ();
-            provider.load_from_resource ("/com/github/lainsce/coin/stylesheet.css");
-            Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
-            set_keep_below (true);
             stick ();
 
             var settings = AppSettings.get_default ();
-- 
2.18.0

